src/Application.php

```diff
@@ -17,23 +17,12 @@
 class Application
 {
     private $application;
-    private $buildContext;
+    private $buildConfig;
 
     public function __construct(string $symfonyVersion)
```

Comment:
We don't need this argument anymore, right? Or if we do, we should pass it directly into `BuildConfig` to set its version (which currently has a hardcoded default of 4.4)?